src/com/facebook/buck/apple/AppleLibraryDescription.java

```diff
@@ -761,8 +688,7 @@ private HeaderSymlinkTree createUnderlyingModuleSymlinkTreeBuildRule(
       }
     }
 
-    if (metadataClass.isAssignableFrom(FrameworkDependencies.class)
-        && buildTarget.getFlavors().contains(AppleDescriptions.FRAMEWORK_FLAVOR)) {
```

Comment:
@robbertvanginkel Shouldn't we just delete the whole `if` branch, as if `buildTarget.getFlavors().contains(AppleDescriptions.FRAMEWORK_FLAVOR` will always be `false`?

Reply:
We could delete the whole branch, but the reason I left it is because the code to copy frameworks into the final bundle will be useful for collecting prebuilt frameworks correctly in a later pr.